### PR TITLE
Feature/integ 622: Add AWS Build button to Sidebar Location

### PR DESCRIPTION
**Pull request by pkeavenycontentful**
## Purpose

Content editors will need a way to trigger AWS Amplify builds from the sidebar. This PR adds the Build button to the sidebar location. 

## Approach
For testing purposes and to get around the CORS errors thrown from Amplify, I used [cors-anywhere](https://www.npmjs.com/package/cors-anywhere) to proxy the request from localhost to a CORS enabled server. This is meant only for development purposes. 

## Testing steps

- Create an AWS amplify app and point it to your localhost
- Go to any Content screen, verify Build button shows up in the sidebar
- Execute Build button, ensure appropriate error messaging occurs
- To test success messaging, follow instructions in the Readme to enable `cors-anywhere`

### CODE REVIEW

File: apps/aws-amplify/src/locations/Sidebar.tsx [hunk 1]
```diff
@@ -1,17 +1,81 @@
...
      To use the cma, inject it as follows.
      If it is not needed, you can remove the next line.
   */
   // const cma = useCMA();
 
-  return <Paragraph>Hello Sidebar Component (AppId: {sdk.ids.app})</Paragraph>;
+  const buildAmplifyApp = () => {
+    setIsLoading(true);
+    const webhookURL =
+      'https://webhooks.amplify.us-east-1.amazonaws.com/prod/webhooks?id=7a31abcf-724e-4072-9329-8541f4e3bb02&token=Lba6wO2uyDKgDA6BmHcqUqXefzVZa9GG8r23HNQVuc';
```
Comment (ryunsong-contentful) [inline apps/aws-amplify/src/locations/Sidebar.tsx:43]: An improvement here would be to place in this a .env file.

Even better yet, just have it call it from the parameters from the Config file using the Sidebar SDK - Matt will know how to do this if you ask him

**Review by matthew-gordon — APPROVED** [review]
LGTM ✅ , I think the files should clear up when https://github.com/contentful/apps/pull/3721 is merged 🤔 and rebase thise

**Review by matthew-gordon — CHANGES_REQUESTED** [review]
Lets look at this together after rebasing and updated the invocation logic, I think we can remove the `cors-anywhere` workaround and such 💪 #3721 is merged

File: apps/aws-amplify/src/locations/Sidebar.tsx [hunk 2]
```diff
@@ -1,17 +1,59 @@
...
   /*
      To use the cma, inject it as follows.
      If it is not needed, you can remove the next line.
   */
   // const cma = useCMA();
 
-  return <Paragraph>Hello Sidebar Component (AppId: {sdk.ids.app})</Paragraph>;
+  const buildAmplifyApp = () => {
+    setIsLoading(true);
+    const webhookURL = 'https://sdlb4gygrnzr252nrx4h3tghym0fameu.lambda-url.us-west-2.on.aws/';
```
Comment (matthew-gordon) [inline apps/aws-amplify/src/locations/Sidebar.tsx:19]: 💪 nice! I know this probably a temp, maybe we shove move into an ENV var? Can do that as cleanup in a "final pass" fast follow with the other small things

**Pull request merged**